vendors/marvell/boards/mw300_rd/aws_demos/application_code/main.c

```diff
@@ -171,13 +155,13 @@ int main( void )
     /* FreeRTOS+TCP stack init */
     vApplicationIPInit();
 
-    /* Create the task to run unit tests. */
-    xTaskCreate( vStartupHook,
-                 "Startup Hook",
-		             mainSTARTUP_TASK_STACK_SIZE,
-                 NULL,
-                 tskIDLE_PRIORITY + 2,
-                 NULL );
+    /* A simple example to demonstrate key and certificate provisioning in
+     * flash using PKCS#11 interface. This should be replaced
+     * by production ready key provisioning mechanism. */
+    vDevModeKeyProvisioning();
```

Comment:
Previously the provisioning was done in a startup task, does it make sense to still run this in the startup task?

Reply:
Moved it back to the startup task to avoid any stack size issues.